contentful/sync_page.py

```diff
@@ -38,7 +41,9 @@ def next(self, client):
         return client.sync({'sync_token': self.next_sync_token})
 
     def _get_sync_token(self):
-        return sub(r'.*sync_token=', '', self.next_sync_url)
```

Comment:
Thanks for the feedback :) I agree this could be changed to `return sub(r'.*sync_token=', '', self.next_sync_url or self.next_page_url)` but that seems a bit brittle. It assumes that the URL will *always* end with `sync_token=<the_sync_token>`. What happens if the API changes and the value of `nextSyncUrl` or `nextPageUrl` is a URL with multiple query string parameters not ending in `sync_token=MKpLD7ChcKHd...`?

For example, the API returns something like`  {"nextPageUrl": "https://cdn.contentful.com/spaces/o46ru064noqc/sync?sync_token=wonDrcKnRgcSOF4&timestamp=1498558724"}`. Using the regex substitution above would break whereas using urllib to extract the correct parameter makes it resilient to these kinds of changes for very little effort and avoids a potential bug in the future.

What do you think?